src/network/networkd-ipv6ll.c

```diff
@@ -116,6 +116,65 @@ int ipv6ll_addrgen_mode_fill_message(sd_netlink_message *message, IPv6LinkLocalA
         return 0;
 }
 
+int link_update_ipv6ll_addrgen_mode(Link *link, sd_netlink_message *message) {
+        uint8_t mode;
+        int family, r;
+
+        assert(link);
+        assert(message);
+
+        r = sd_rtnl_message_get_family(message, &family);
+        if (r < 0)
+                return r;
+
+        if (family != AF_UNSPEC)
```

Comment:
Is this intended to be AF_UNSPEC instead of AF_INET6?

Reply:
`AF_UNSPEC` here is correct. Here, `family` may be `AF_BRIDGE` in some message. This intends to avoid such case.